app/views/ontologies/concepts_browsers/_concepts_tree.html.haml

```diff
@@ -16,6 +16,4 @@
                         'chosen-enable-colors-value': 'true'}}
   -# Class tree
   %div#sd_content.card.p-1.py-3{style: 'overflow-y: scroll; height: 60vh;'}
-    = render TurboFrameComponent.new(id: 'concepts_tree_view',
-                                     src: "/ajax/classes/treeview?ontology=#{@ontology.acronym}&conceptid=#{escape(@concept.id)}&concept_schemes=#{params[:concept_schemes]}&auto_click=false&language=#{request_lang}",
```

Comment:
think to remove the action behind

Reply:
Still to do;

The thing to here is to clean the code that was used before:
* in routes.rb, remove  `/ajax/classes/treeview`
* in concept_controller.rb, remove  the action `show_tree`

but before doing that, you need to be sure that it is no longer used anywhere.